src/app-layout/__tests__/runtime-drawers.test.tsx

```diff
@@ -1183,9 +1198,34 @@ describe('toolbar mode only features', () => {
 
         findDrawerTriggerById('global-drawer', renderProps)!.click();
         expect(onToggle).toHaveBeenCalledWith({ isOpen: true, initiatedByUserAction: true });
-        renderProps.globalDrawersWrapper.findCloseButtonByActiveDrawerId('global-drawer')!.click();
+        findCloseButtonByActiveDrawerId('global-drawer', renderProps)!.click();
         expect(onToggle).toHaveBeenCalledWith({ isOpen: false, initiatedByUserAction: true });
       });
+
+      (type === 'global-ai' ? test : test.skip)(
```

Comment:
I started a new file for this new API: https://github.com/cloudscape-design/components/blob/main/src/app-layout/__tests__/runtime-drawers-widgetized.test.tsx

It is better to add new exclusive features there